docker.mk

```diff
@@ -11,7 +11,7 @@ REGISTRY_IMG_NAME = netbricks
 LINUX_HEADERS = -v /lib/modules:/lib/modules -v /usr/src:/usr/src
 DPDK_VER = 17.08
 BASE_DIR ?= $(or $(shell pwd),~/occam/Netbricks)
-MAX_CORES ?= 1
+MAX_CORES ?= 3
```

Comment:
maybe remove this the. However, not sure why cpuset-cpus falters here, according to the docs: https://docs.docker.com/engine/reference/run/#cpuset-constraint? @drunkirishcoder

Reply:
failed because travis runner didn't have access to 4 cores.

Reply:
gotcha. boom.